src/StackAdmin/Azs.Compute.Admin/Module/Azs.Compute.Admin/Generated.PowerShell.Commands/SwaggerDefinitionCommands/New-QuotaCustomObject.ps1

```diff
@@ -0,0 +1,115 @@
+<#
```

Comment:
The filename should be ConvertTo-SDKQuota.ps1 to match function name (I suggest change below)

Reply:
done